UI rework towards Contribution Hub

comzeradd opened the pull request:
UI reworked since this is now more than a FoxFooding platform. I removed the big sidebar; having essential content and specific 'call to action' we can get more focused user experience.

comzeradd committed:
UI rework towards Contribution Hub

comzeradd commented:
@akatsoulas r?

akatsoulas commented:
![screen shot 2015-10-14 at 12 40 27](https://cloud.githubusercontent.com/assets/900565/10480085/f36091c8-7270-11e5-824c-251f7a88afcb.png)

A few thoughts/comments:
- Footer seems to be out of place.
- The fxos-wordmark seems too big to me.
- The same goes to homescreen. Plus it seems a bit funny since only half of the image is visible.
- Mobile is broken.

akatsoulas commented:
The images are missing from the 'devices' section.

akatsoulas commented:
I would suggest to leave the dashboard in place until decided otherwise.

akatsoulas commented:
Please provide a better, more descriptive alt description to all images.

comzeradd commented:
@akatsoulas PR updated

I did some html/css changes:
- Added dashboard as a navigation link
- Removed broken image from devices. We should add these when we know which devices.
- Added missing alt to some images
- Tweak a bit the footer, but without content it's not easy to fix it at the bottom for all resolutions.
- I kept the homescreen image for the tablet resolutions too (the one on your screenshot)

akatsoulas commented on feedthefox/base/static/css/app.css:
```diff
@@ -1,42 +1,154 @@
     font-family: 'Fira Sans';
+    src: url('../fonts/FiraSans-Regular.eot');
+    src: url('../fonts/FiraSans-Regular.eot?#iefix') format('embedded-opentype'),
+         url('../fonts/FiraSans-Regular.woff') format('woff'),
```
I would suggest to revert to the absolute path, as it was before.

akatsoulas replied:
Please apply this to any occurrence in the code.

akatsoulas commented on feedthefox/base/static/css/app.css:
```diff
@@ -1,42 +1,154 @@
+    color: #484848;
     font-size: 14px;
     line-height: 1.4;
     font-family: 'Fira Sans';
```
Should we add a default font family here, eg 'sans-serif' ?

comzeradd commented:
@akatsoulas thanks for the comments. PR updated.

akatsoulas commented on .gitignore:
```diff
@@ -1,3 +1,4 @@
 *.py[co]
 .env
 /static
+*.sqlite3
```
Since we are not using this in the project, please remove it from here. You can always ignore this locally.

akatsoulas commented:
What about using a sticky footer?

akatsoulas commented:
Let's merge this and we can fix any issues that may arise as we proceed and have more specs/content. r+wc

comzeradd commented:
Thanks. I fixed `.gitignore` and tweaked a bit the footer to stick at the bottom. We can revisit this when we have content.
